Create: issue templates (#339)
XTLS/Xray-core: .github/ISSUE_TEMPLATE/bug-en.md

---
name: Bug Report
about: Create a bug report of Xray.
title: '[Bug] <bug you are reporting>'
labels: ''
assignees: ''

---

<!-- Thanks for your reporting.

> 1. This issue is used to submit a bug of Xray. If you are not sure, please give feedback in the group.

Reviewer: ```suggestion
1. Please make sure you are submitting a bug of Xray-core. If you are not sure, please give feedback in our telegram group.
```

> 1. Please make sure you are submitting a bug of Xray-core. If you are not sure, please give feedback in our telegram group.

Reviewer: ```suggestion
1. Please make sure you are submitting a bug of Xray-core. issues like "how to use xray-core?" are not accepted, If you are not sure if it is a bug or not, please contact us in our telegram group first.
```

> 1. Please make sure you are submitting a bug of Xray-core. If you are not sure, please give feedback in our telegram group.

Reviewer 2: ```suggestion
1. Please make sure you are submitting a bug of Xray-core rather than a bug of clients such as Qv2ray, v2rayN, v2rayNG, luci-app-ssr-plus, luci-app-passwall. If you are not sure, please give feedback in our telegram group.
```

> 2. Please check the existing issues, discussions and documentation in detail. Issue which report a known bug may be closed.

Reviewer: ```suggestion
2. Please check existing issues and discussions first and read the documentation in detail. If you are submitting a known issue, it will be closed
```

> 2. Please check existing issues and discussions first and read the documentation in detail. If you are submitting a known issue, it will be closed

Reviewer: ```suggestion
2. Please check existing issues and discussions first and read the documentation in detail. Duplicated issues will be closed
```

> 3. Please don't report issue like "I can't use a feature". It's probably your fault.

Reviewer: ```suggestion
```

Reviewer: 意思与第一条重复，且用不了一个功能也可能是BUG导致，不能单纯这么定义

Reviewer 3: 这一条是为了避免“自信的伸手党”使用issue反馈他自己的问题。也许需要修改下措辞，但是一定要保留。

4. You should fully complete the following contents.

> 5. Make sure that ALL of private information is covered up.

Reviewer: ```suggestion
5. Make sure the content you are submitting does not contain your private information.
```

-->

**Describe the bug**

> <!-- A clear and concise description of what the bug is. -->

Reviewer: ```suggestion
<!-- A clear and concise description of what bug you have found. -->
```

**To Reproduce**

> <!-- Steps to reproduce the behavior: -->

Reviewer: ```suggestion
<!-- Steps to reproduce the bug: -->
```

1.
2.
3.
4.

**Expected behavior**

> <!-- A clear and concise description of what you expected to happen. -->

Reviewer: ```suggestion
<!-- A clear and concise description of what you expected would happen. -->
```
